### DISCUSSION TITLE
Requests not being recorded when enabling user profiles and proxies

### DISCUSSION THREAD

**Discussion by sadikhan918** [seleniumbase/SeleniumBase#3086] [Q&A] [answered]
I want to see the requests that are being made when I visit a certain page. I have user_data_dir enabled and set to a path as well as a proxy. I also have wire=True. When I visit the page, the requests list for the driver is empty. This is what my setup looks like:

```
from seleniumbase import SB

with SB(wire=True, user_data_dir="some_user_path". proxy="some_proxy") as sb:
   sb.open(url)
   # This is where the requests should show up
   for request in sb.driver.requests:
      print(request)
      # Nothing is getting printed
      ...

```

When I attempt the same code without specifying the user_data_dir or the proxy, it works fine and all the requests show up normally. The problem is that I need the sessions stored in the user directory and I need the proxy with it.

**Comment by mdmintz** [comment, marked as answer]
Wire Mode uses `selenium-wire` directly. See https://github.com/wkeeling/selenium-wire for details. That repo seems to have lots of issues when changing proxy settings (https://github.com/search?q=repo%3Awkeeling%2Fselenium-wire+proxy&type=issues). Since your issue appears to be directly related to `selenium-wire`, there's nothing that can be done here to fix it.
